hooks/useTokenList.tsx

```diff
@@ -0,0 +1,34 @@
+import { networks } from "@/utils/networks";
+import useSWR from "swr";
```

Comment:
thought we were going to prefer react-query, are we going with swr instead?

Reply:
@alexkeating has experience with react-query, so think we said we're going to try swr on this project since it's supposed to be a bit simpler (and consequently less flexible as app grows) + it's by vercel/next team so should play nicely together

Reply:
Oh sorry I didn't mention this, but along with Matt's reasons above I tried setting up react-query with next and it was causing some errors. Since we weren't strongly opposed to SWR I switched over because there was less friction setting it up.

Reply:
sg!